Sources/NiceComponents/Components/ResizableImage.swift

```diff
@@ -59,26 +47,82 @@ public struct ResizableImage: View {
 
     public var body: some View {
         if let url = url {
-            KFImage(url)
-                .renderingMode(tintColor == nil ? .original : .template)
-                .resizable()
-                .placeholder {
-                    LoadingView(loadingStyle ?? .large)
-                }
-                .foregroundColor(tintColor)
-                .scaledToFill()
-                .frame(width: width, height: height)
-                .clipped()
+            AsyncImage(url: url) { image in
+                image.resizable()
+                    .renderingMode(tintColor == nil ? .original : .template)
+                    .aspectRatio(contentMode: contentMode)
+                    .frame(maxWidth: width, maxHeight: height)
+                    .foregroundColor(tintColor)
+            } placeholder: {
+                placeholderView()
+                    .frame(width: width, height: height)
```

Comment:
Will this cause some weird resizing if the image that ends up rendering is smaller than the maxWidth/Height?

Reply:
Yeah I was thinking this as well but didn't test it. Will test / take a look